setup/includes/upgrades/common/3.0.0-update-tvs-list-legacy.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * Common upgrade script for modify modTemplateVar with 'list-multiple-legacy' input type
+ *
+ * @var modX $modx
+ * @package setup
+ */
+
+use MODX\Revolution\modTemplateVar;
+
+/** @var modTemplateVar $tvs_list_legacy */
+$tvs_list_legacy = $modx->getCollection(modTemplateVar::class, ['type' => 'list-multiple-legacy']);
+
+if ($tvs_list_legacy) {
```

Comment:
Make sense to check it first by `$modx->getCount(modTemplateVar::class, $condition)` and after do replacing. Just to avoid wrapping code in the if statement.